spec/models/device_problem_spec.rb

```diff
@@ -4,8 +4,14 @@
 
   it { should belong_to(:device) }
 
+  let!(:image) do
+    Photo.create(name: 'Roast Image') do |photo|
```

Comment:
If this works, I'd change all the places you're using the block form of `create` in favor of the non-block form.